### PR TITLE
Don't capitalize Mr, Ms, Mrs

### PR DESCRIPTION
**Pull request by GurraB**
The current version will capitalize Mr, Mrs, Ms. I added a regex and an if-statement to handle them separately.

Previous
`Mr Mrs Ms -> MR MRS MS`

Now
`Mr Mrs Ms -> Mr Mrs Ms`

### CODE REVIEW

File: titlecase/tests.py [hunk 1]
```diff
@@ -303,6 +303,30 @@
...
+        "Ms",
+    ),
+    (
+        "Mss",
+        "MSS",
+    ),
+    (
+        "Mr Person",
+        "Mr Person",
+    ),
```
Comment (ppannuto) [inline titlecase/tests.py:329]:
```suggestion
    # https://github.com/ppannuto/python-titlecase/pull/67
    (
        "Mr mr Mrs Ms Mss , Mr. and Mrs. Person",
        "Mr Mr Mrs Ms MSS , Mr. and Mrs. Person",
    ),
```

Comment (GurraB) [reply titlecase/tests.py]: Yes, that's much better. I don't know why I didn't think of that :D

**Comment by GurraB** [comment]
So the `and` was made uppercase when using a dot. I'm guessing it's treated as a new sentence. The word after Mr, Mrs or Ms should always be uppercase I suppose. Do you think I should add that we uppercase the first char in the next word automatically when the word is Mr, Mrs or Ms even if there is not a dot?

**Comment by ppannuto** [comment]
Ugh, this is what is unfortunate about a regex-based approach, you keep running into corner cases. From a quick survey of style guides online, everything after a `Mr/Mrs/Dr` should be capitalized _except_ `and`.

If you want to add a special case so the library handles it correctly, that'd be awesome. If you'd rather just change the test to not include an `and` case it would just be return to what it was already doing anyway, so no regression, and would be fine with me.

**Comment by GurraB** [comment]
I can't figure out a smart way to add exception for `and` without butchering the code. I'm fine with it as is now, let me know if you changed your mind and want changes.

**Pull request merged**